File: Domain/TransApp.Domain/Addresses/AddressAvailabilityModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TransApp.Domain.Addresses
{
    public class AddressAvailabilityModel
    {

        public int Id { get; set; }

        public int? AddressId { get; set; }
        public bool IsClosed { get; set; }

        public int? Day { get; set; }

        public string AmStart { get; set; }

        public string AmStop { get; set; }

        public string PmStart { get; set; }

        public string PmStop { get; set; }

        public int? UserIdCreated { get; set; }

        public DateTime? DateCreated { get; set; }

        public int? UserIdModified { get; set; }

        public DateTime? DateModified { get; set; }
    }
}

File: Site/Presentation/TransApp.Site.FrontEnd/src/app/shared/common/models/address-availability-model.ts
export class AddressAvailabilityModel {
    public id: number;
    public addressId: number;
    public day: number;

    public isClosed : boolean;

    public amStart: string;
    public amStop: string;
    public pmStart: string;
    public pmStop: string;

    public userIdCreated: number;
    public dateCreated: Date;
    public userIdModified: number;
    public dateModified: Date;
}